cumulus/polkadot-parachain/Cargo.toml

```diff
@@ -118,10 +118,13 @@ tokio = { version = "1.32.0", features = ["macros", "parking_lot", "time"] }
 wait-timeout = "0.2"
 
 [features]
-default = []
+default = [
+	"beacon-spec-mainnet",
```

Comment:
@claravanstaden @alistair-singh I am not sure that this is a good thing to set it as default in `polkadot-parachain` binary, because it is used for other parachains not related to any bridge stuff, can you, please, check if this PR will work: https://github.com/paritytech/polkadot-sdk/pull/2777 ?

Reply:
@claravanstaden @alistair-singh @vgeddes 
also I think we should make it `beacon-spec-mainnet` as production-like by default without need to even specify at runtime `Cargo.toml`, and if you need that `minimal` stuff for testing or whatever, that should be allowed on-demand by feature like `beacon-spec-minimal` or `beacon-spec-fast-runtime` (`polkadot-sdk` uses `fast-runtime`)

Reply:
We're working on this - a complication in our environment is that the `fast-runtime` / `beacon-spec-minimal` is incompatible with `runtime-benchmarks`, or in other words, we can only compile runtime benchmarks with the the full production mainnet spec for Ethereum.

We're busy looking for a solution, its just an engineering problem...